packages/ckeditor5-list/src/listproperties/ui/listpropertiesview.ts

```diff
@@ -366,6 +366,7 @@ export default class ListPropertiesView extends View {
 			const startIndex = inputElement.valueAsNumber;
 
 			if ( Number.isNaN( startIndex ) ) {
+				startIndexFieldView.errorText = t( 'Invalid start index value.' );
```

Comment:
I would add a comment explaining the cases/values that could result in this error.